input/docs/handbook/data-binding/index.md

```diff
@@ -31,109 +21,96 @@ implement it differently:
   changes.
 
 * **Xaml-based:** - Implement `IViewFor<T>` by hand and ensure that ViewModel
-  is a `DependencyProperty`.
+  is a `DependencyProperty`. Additionally, for `UserControl`s you can use a base 
+  class that encapsulates `IViewFor` behavior - `ReactiveUserControl<TViewModel>`.
   
 For a detailed overview of the bindings on each platform, see the "Binding" section.
 
-### Types of Bindings
+# Types of Bindings
 
-Once you implement `IViewFor<T>`, binding methods are now available as
-extension methods on your class. Like many other things in ReactiveUI, you
-should only set up bindings in a constructor or setup method when the view is
-created.
+Once you implement `IViewFor<T>`, binding methods are now available as extension methods on your class. Like many other things in ReactiveUI, you should only set up bindings in a constructor or even better in a [WhenActivated](../when-activated) block to enable [activation and deactivation](../when-activated) feature for your Views and associated ViewModels.
```

Comment:
Should probably mention the memory leaks situation for dependency property based properties or link to the documentation about it